examples/angular-cli/src/stories/core/parameters/bootstrap-options.stories.ts

```diff
@@ -0,0 +1,24 @@
+import { Meta, Story } from '@storybook/angular/types-6-0';
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'component-with-whitespace',
+  template: ` <div>
+    <p>Some content</p>
+  </div>`,
+})
+class ComponentWithWhitespace {}
+
+export default {
+  title: 'Core / Parameters / With Bootstrap Options',
+  parameters: {
+    bootstrapOptions: {
```

Comment:
I know it's merged, but I noticed it should be `bootstrapModuleOptions` probably.

Reply:
cc @ThibaudAV

Reply:
yes I think so 😁